src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,96 @@
+const expensesService = require('../services/expenses.service');
+const usersService = require('../services/users.service');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  if (!userId && !categories && !from && !to) {
```

Comment:
Why do you need this check? If you want to sent 400 error consider using middleware for validation purpose